chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRequestScopeJmxEndpointTest.java

```diff
@@ -104,13 +105,17 @@ void isChaosMonkeyActive() {
 
   @Test
   void enableChaosMonkey() {
-    assertThat(chaosMonkeyJmxEndpoint.enableChaosMonkey(), startsWith("Chaos Monkey is enabled"));
+    ZonedDateTime enabledAt = ZonedDateTime.now();
+    ZonedDateTime timeReturned = chaosMonkeyJmxEndpoint.enableChaosMonkey().getEnabledAt();
+    assertTrue(timeReturned.isAfter(enabledAt) || timeReturned.isEqual(enabledAt));
     assertThat(chaosMonkeySettings.getChaosMonkeyProperties().isEnabled(), is(true));
   }
 
   @Test
   void disableChaosMonkey() {
-    assertThat(chaosMonkeyJmxEndpoint.disableChaosMonkey(), startsWith("Chaos Monkey is disabled"));
+    ZonedDateTime disabledAt = ZonedDateTime.now();
+    ZonedDateTime timeReturned = chaosMonkeyJmxEndpoint.disableChaosMonkey().getDisabledAt();
+    assertTrue(timeReturned.isAfter(disabledAt) || timeReturned.isEqual(disabledAt));
```

Comment:
The reason I used isAfter and isEqual is because every time I ran the test, except one time, it had isAfter false and isEqual true! So just in case others encounter that odd event, I didn't want them to get a failed test!